src/components/dialogs/load-creation-dialog.js

```diff
@@ -0,0 +1,483 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, { useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import PropTypes from 'prop-types';
+import {
+    InputLabel,
+    MenuItem,
+    Popper,
+    Select,
+    TextField,
+} from '@material-ui/core';
+import FormControl from '@material-ui/core/FormControl';
+import { Autocomplete, createFilterOptions } from '@material-ui/lab';
+import TextFieldWithAdornment from '../util/text-field-with-adornment';
+import { useParams } from 'react-router-dom';
+import {
+    createLoad,
+    fetchBusbarSectionsForVoltageLevel,
+    fetchBusesForVoltageLevel,
+} from '../../utils/rest-api';
+import {
+    displayErrorMessageWithSnackbar,
+    useIntlRef,
+} from '../../utils/messages';
+import { useSnackbar } from 'notistack';
+import { validateField } from '../util/validation-functions';
+import { makeStyles } from '@material-ui/core/styles';
+
+const filter = createFilterOptions();
+
+const useStyles = makeStyles((theme) => ({
+    helperText: {
+        margin: 0,
+    },
+    popper: {
+        style: {
+            width: 'fit-content',
+        },
+    },
+}));
+
+/**
+ * Dialog to create a load in the network
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param {String} title Title of the dialog
+ */
+const LoadCreationDialog = ({ open, onClose, network }) => {
+    const classes = useStyles();
+
+    const studyUuid = decodeURIComponent(useParams().studyUuid);
+
+    const intl = useIntl();
+    const intlRef = useIntlRef();
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const [busOrBusbarSectionOptions, setBusOrBusbarSectionOptions] = useState(
+        []
+    );
+
+    const [loadId, setLoadId] = useState('');
+
+    const [loadName, setLoadName] = useState('');
+
+    const [loadType, setLoadType] = useState('');
+
+    const [activePower, setActivePower] = useState('');
+
+    const [reactivePower, setReactivePower] = useState('');
+
+    const [voltageLevel, setVoltageLevel] = useState('');
```

Comment:
`useState(null); `?

Reply:
My bad, voltageLevel is an object, so it has to be set to null (so is busOrBusbarSection). I modified all initializations of voltageLevel and busOrBusbarSection to `null` instead of `''`.